pkg/render/manager.go

```diff
@@ -77,10 +81,12 @@ func Manager(
 	oidcConfig *corev1.ConfigMap,
 	management bool,
 	tunnelSecret *corev1.Secret,
+	internalTrafficSecret *corev1.Secret,
 ) (Component, error) {
 	tlsSecrets := []*corev1.Secret{}
+	tlsAnnotations := make(map[string]string)
 
-	if tlsKeyPair == nil && !management {
```

Comment:
Hey @asincu do you remember why there was a check for whether or not it was a management cluster? 

Just want to understand what scenario it was accounting for and why it's no longer needed.

Reply:
Yup, when the cert was shared, it was created by the kube-controller renderer.